core/lib/spree/testing_support/factories/address_factory.rb

```diff
@@ -11,7 +11,7 @@
     end
 
     firstname 'John'
-    lastname 'Doe'
+    lastname nil
```

Comment:
@gmacdougall I can see what you mean, and I hate to break specs, but we'd rather break their specs than their app, right?  E.g., we're trying this out on our own codebase and having the factory change helped us find a couple extra places where we weren't handling a possibly-nil last name.  In general we want factories not to set attributes that aren't actually required and this attribute is no longer required right?